test/integration/client_test.cpp

```diff
@@ -216,13 +221,17 @@ TEST_F(ClientServerTest, SendQueryWhenStatelessInvalid) {
 }
 
 TEST_F(ClientServerTest, SendQueryWhenValid) {
+  // TODO: 30/04/2018 x3medima17, fix Uninteresting mock function call, IR-1187
```

Comment:
Is this behavior appears only in your PR? If that, you have to fix it in the current PR.

Reply:
Nope, I checked this test in develop, it has the same behaviour there